Maquetado/scss/footer.scss

```diff
@@ -0,0 +1,24 @@
+footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-flow: column nowrap;
+  padding:50px 15px 15px 10px;
```

Comment:
there should be a space after the `:` you should also use the same paddings at each side in order to have the elements centered inside